featuremanagement/_featuremanager.py

```diff
@@ -341,7 +360,7 @@ def _check_feature(self, feature_flag_id, targeting_context, **kwargs):
             evaluation_event.feature = feature_flag
             return evaluation_event
 
-        evaluation_event = self._check_feature_filters(feature_flag, targeting_context, **kwargs)
+        evaluation_event = self._check_feature_filters(feature_flag, evaluation_event, targeting_context, **kwargs)
```

Comment:
Eventually, feature_flag will be set to evaluation_event.feature. And feature_flag will not be modified in `_check_feature_filters`, let's set `evaluation_event.feature = featuer_flag` in advance and remove `feature_flag` from the parameters of `_check_feature_filters`.

Currently, EvaluationEvent.Feature is never used.

Reply:
I think we are including this as part of the schema as we provide the customer the evaluation event if they want. DotNet does the same thing https://github.com/microsoft/FeatureManagement-Dotnet/blob/preview/src/Microsoft.FeatureManagement/FeatureDefinition.cs. @rossgrambo?

Reply:
nvm, figured out what you meant. Should be updated.